toolset/run-tests.py

```diff
@@ -189,6 +189,17 @@ def main(argv=None):
         default=None,
         help='The network mode to run docker in')
 
+    parser.add_argument(
+        '--local-dev-mode',
+        action='store_true',
+        default=False,
+        help='server and database both run locally, very easy during development, building containers takes huge aount of time every single time if there are changes in framework source code')
+    parser.add_argument(
+        '--enable-core-dumps',
+        action='store_true',
+        default=False,
+        help='sometimes we may need core dumps from the conatiner')
```

Comment:
This sounds interesting.  Where/how do we look at the core dumps?

Reply:
You could probably get inside the container with privileged mode on and then install gdb once in and then look at the dumps there or could also copy that back to the host and analyze there.